### PR TITLE
setup.py cleanup

### PR DESCRIPTION
**Pull request by jjshoots**
# Description

This updates the setup.py file to be consistent with that of PettingZoo.
The list of changes are as follows:
- Move version into gymnasium/__init__.py
- Updated license
- Edit setup to be in line with that of pettingzoo

### CODE REVIEW

**Review by pseudo-rnd-thoughts — COMMENTED** [review]
Why the need to change the gymnasium setup.py or change version. This seemed like simple and effective method of doing it all

**Comment by jjshoots** [comment]
Ready for review @pseudo-rnd-thoughts

File: setup.py [hunk 1]
```diff
@@ -51,33 +55,23 @@ def get_version():
     set(itertools.chain.from_iterable(map(lambda group: extras[group], all_groups)))
 )
 
+version = get_version()
```
Comment (pseudo-rnd-thoughts) [inline setup.py:58]: This function doesn't seemed to be used and the setup version=get_version

**Review by pseudo-rnd-thoughts — COMMENTED** [review]
Generally looks good, just a comment on the version variable

File: setup.py (hunk 1 above)
Comment (jjshoots) [reply setup.py:58]: Fixed to do `version=version`.

**Review by pseudo-rnd-thoughts — APPROVED** [review]

**Pull request merged**